wiki/People/The_Team/Developers/id.md

```diff
@@ -41,9 +37,8 @@ Upaya pengembangan osu! dipimpin oleh ![][flag_AU] **[peppy](https://osu.ppy.sh/
 ### Desainer
 
 - ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765)
-- ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202)
 
-### Community Manager
+### Manajer komunitas
```

Comment:
o i'm not sure whether `Manajer` is a proper word or not tbh :(

try `Pengelola Komunitas` maybe? sounds much better either way imo

Reply:
yeah `Pengelola komunitas` is better tbh the only non-people `manajer` that I've ever heard is `manajer keuangan`